From 6569c555ef6b936e96b454b4e9363804d324586b Mon Sep 17 00:00:00 2001
From: huyIttos2 <huyittos2071195@gmail.com>
Date: Mon, 13 Jan 2020 23:32:56 -0800
Subject: [PATCH] modified directorystructure

---
 src/info.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/info.php b/src/info.php
index a428e10..b610f7a 100644
--- a/src/info.php
+++ b/src/info.php
@@ -1,4 +1,4 @@
-
+hello jerry
 #include <stdio.h>
 +size_t my_strlen(char *s)
 {
-- 
2.17.1

